### PR TITLE
Enhancement/s5menu ui

### PR DESCRIPTION
**Pull request by davidberard2**
This pull request insure icon color is white by refactoring the file type.  Still need to fix many small issue.

### CODE REVIEW

File: app/src/main/res/drawable/ic_star_white_24dp.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="@color/white"
+        android:pathData="M12,17.27L18.18,21l-1.64,-7.03L22,9.24l-7.19,-0.61L12,2 9.19,8.63 2,9.24l5.46,4.73L5.82,21z"/>
+</vector>
```
Comment (Alex-Hayeur) [inline app/src/main/res/drawable/ic_star_white_24dp.xml:9]: I haven't seen this way of editing/creating Android icons, so thank you for that.

**Review by Alex-Hayeur — APPROVED** [review]
A small fix but great for UI and much simpler than what I had thought to do. Good code and much appreciated.

File: app/src/main/res/drawable/ic_star_white_24dp.xml (hunk 1 above)
Comment (davidberard2) [reply app/src/main/res/drawable/ic_star_white_24dp.xml:9]: no problem, it is called vector method. it allows you to scale it to any dimension and change the properties of it. It is a little more stable.

**Review by kwreen — APPROVED** [review]
Solid way of displaying the icons from now on

**Pull request merged**